modin/data_management/query_compiler/pandas_query_compiler.py

```diff
@@ -1112,12 +1112,13 @@ def to_sql(self, **kwargs):
         columns = self.columns
 
         def func(df, **kwargs):
+            raise ValueError(kwargs)
```

Comment:
These changes are irrelevant. This method can go away if you accept the `PandasOnRayIO` changes.